README.md

```diff
@@ -2,30 +2,45 @@
 
 A React Native Camera and Player for MediaPipe applications.
 
-## Installation
+* [Documentation](https://cdiddy77.github.io/react-native-mediapipe/)
 
+## Requirements
+* Requirements go here.
+
+## Features
+* 🎥 Video streaming from a react-native app to a media pipe
+* 🔍 AI Object Detection
+* 🧩 Works as a react-native component
+
+Install mediapipe from npm:
 ```sh
 npm install react-native-mediapipe
 ```
 
-## Usage
+## Example
 
 ```js
-import { MediapipeView } from "react-native-mediapipe";
+import { MediaPipe_Camera_ } from "react-native-mediapipe";
```

Comment:
should be `MediaPipeCamera`

Reply:
also, in order to use the package, you have to install/add `react-native-vision-camera` and `react-native-worklets-core` per the video

Reply:
Also, you need to add documentation for the parts of [the video](https://www.loom.com/share/97548713a7f54114be4e00c9fa335fdc) from 5:43 to 12:40

- modify `babel.config.js`
- modify the `build.gradle` file
- modify the iOS and android manifest files to request camera permissions

and also 
- cd ios && bundle install && pod install

All of these steps are documented in the [Vision Camera documentation](https://react-native-vision-camera.com/docs/guides#updating-manifests)

Once the user is done with all that, there is more configuration that they need to do, but that will vary from project to project and so is probably left to the real documentation

Reply:
I added all of the following steps and added code snippets to make sure that they're there as well as notifying that bundle install only needs to be done once.